README.md

```diff
@@ -1 +1,69 @@
-# mobile-menu
\ No newline at end of file
+# mobile-menu
+
+![Microverse](https://img.shields.io/badge/-Microverse-6F23FF?style=for-the-badge)
+
+## Description
+
+This is a simple `Portfolio` project to demonstrate coding best practices including but not limited to understanding Gitflow, creating different base branches, understanding version control, and writing descriptive PRs and commit messages..
+
+
+## Library Directory 📙
+
+| Contents                    |
+| --------------------------- |
+| [Screenshot](#screenshot)   |
+| [Built With](#built-with-🛠) |
+| [Getting Started](#getting-started-🛠) |
+| [Contributing](#contributing🛠) |
+| [Authors](#authors)         |
+| [License](#license)         |
+
+
+ ## Screenshot
+
+![img](./mockup)
```

Comment:
- [ ] [OPTIONAL] Please fix this broken link! I think it would be better to make sure your screenshot exists before committing, this allows you to be professional!